docs/design/2022-08-31-placement-rules-in-SQL-works-with-tables-with-TiFlash-replica.md

```diff
@@ -0,0 +1,29 @@
+# Placement rules in SQL works with tables with TiFlash replica
```

Comment:
Currently we do not have a linter rule in CI for markdowns. Prefer to install the "Prettier" extension in VsCode and use it to format your markdown.